package/skelcd-control-openSUSE.spec

```diff
@@ -39,7 +39,7 @@ BuildRequires:  libxml2-tools
 # xsltproc
 BuildRequires:  libxslt-tools
 # Added 'lsm' section (jsc#SLE-22069)
```

Comment:
This comment refers to the reason for requiring that version of yast2-installation-control. Now it's outdated.